### PR TITLE
Issue #1375 fix duplicated entries ipf

### LINKED ISSUE #1375: LHM conversion: Water balance of wells
In my conversion of the LHM, I notice that the total water balance of wells differs a lot between the iMOD5 conversion and iMOD Python conversion.

In the iMOD5 conversion 1.4e6 m3/d is injected and 4.3e6 m3/d is extracted, whereas in iMOD Python's 8e6 m3/d is injected and 1.5e6 m3/d is extracted. I expect more to be extracted than injected, so iMOD5's seems to behave better.

I first thought this was because iMOD5 and MODFLOW6 use different sign conventions, but this doesn't appear to be the case:

From iMOD 5 manual:

> Extraction rate, negative rates are extractions, positive rates injections

From MODFLOW6 manual:

> Negative values of QWEL are used to indicate well discharge (pumping), whereas positive values of QWEL indicate 
   an injection well

Things to investigate:
- Are rates similarly distributed?
- Are the same wells filtered?
- Differences between text files?

### PR DESCRIPTION
**Pull request by JoerivanEngelen**
Fixes #1375

# Description
``imod.mf6.Well.from_imod5_data`` and  ``imod.mf6.LayeredWell.from_imod5_data`` now sum rates of duplicated well entries, so entries with duplicate x, y, and depth information instead of taking the values of the first entry.

Also adds some additional test for duplicate entries in wells without associated files, which were lacking.

# Checklist

- [x] Links to correct issue
- [x] Update changelog, if changes affect users
- [x] PR title starts with ``Issue #nr``, e.g. ``Issue #737``
- [x] Unit tests were added
- [ ] **If feature added**: Added/extended example

### CODE REVIEW

File: docs/api/changelog.rst [hunk 1]
```diff
@@ -43,6 +43,10 @@ Fixed
 - :meth:`imod.mf6.Well.from_imod5_data` and
   :meth:`imod.mf6.LayeredWell.from_imod5_data` ignore well rates preceding first
   element of ``times``.
+- :meth:`imod.mf6.Well.from_imod5_data` and
+  :meth:`imod.mf6.LayeredWell.from_imod5_data` now sum rates of duplicated well
+  entries, so entries with duplicate x, y, and depth information instead of
+  taking the values of the first entry.
```
Comment (Manangka) [inline docs/api/changelog.rst:49]: What about:
 :meth:`imod.mf6.Well.from_imod5_data` and
  :meth:`imod.mf6.LayeredWell.from_imod5_data` now sum the  rates of well
  entries that are on the same location ( same x, y, and depth) instead of
  taking the values of the first entry.

File: imod/mf6/wel.py [hunk 2]
```diff
@@ -691,6 +694,8 @@ def from_imod5_data(
 
         # Unpack wel indices by zipping
         varnames = [("x", float), ("y", float)] + cls._depth_colnames
+        # if pkg_data["has_associated"]:
+        #     varnames += [("id", str)]
```
Comment (Manangka) [inline imod/mf6/wel.py:698]: This can be removed

**Review by Manangka — APPROVED** [review]

File: docs/api/changelog.rst (hunk 1 above)
Comment (JoerivanEngelen) [reply docs/api/changelog.rst:49]: Done

File: imod/mf6/wel.py (hunk 2 above)
Comment (JoerivanEngelen) [reply imod/mf6/wel.py:698]: Woops, nice catch! That was some faulty code I introduced and commented out.
